build/commands/lib/config.js

```diff
@@ -165,6 +165,11 @@ const Config = function () {
   this.isCI = process.env.BUILD_ID !== undefined
   this.braveStatsApiKey = getNPMConfig(['brave_stats_api_key']) || ''
   this.braveStatsUpdaterUrl = getNPMConfig(['brave_stats_updater_url']) || ''
+  this.p3aJsonUploadUrl = getNPMConfig(['p3a_json_upload_url']) || 'https://p3a-json.brave.com/'
```

Comment:
we don't include defaults here because we don't want non-Brave builds to use our servers

Reply:
Prior to this patch we hard-coded the url in brave_p3a_service.cc, relying on the channel definition to distinguish submissions from unofficial builds. Could we go back that here, and move the npm.config hooks to a separate issue if the previous design is counter to policy?